gtk4/src/expression.rs

```diff
@@ -182,6 +246,46 @@ impl glib::value::ToValueOptional for Expression {
     }
 }
 
+// rustdoc-stripper-ignore-next
+/// Trait containing convenience methods in creating [`gtk::PropertyExpression`] that
+/// looks up a property of a [`glib::Object`].
+///
+/// # Example
+///
+/// `label_expression` is a [`gtk::Expression`] that looks up at Button's `label`
+/// property.
+///
+/// ```no_run
+/// # use gtk4 as gtk;
+/// use gtk::prelude::*;
+///
+/// let button = gtk::Button::new();
+/// button.set_label("Label property");
+///
+/// let label_expression = button.property_expression("label");
+/// ```
+pub trait GObjectPropertyExpressionExt {
```

Comment:
Does it make sense to rename this to `PropertyExpressionExt`?

Reply:
Not, it's not an extension trait for `PropertyExpression` but one for `glib::Object`